packages/devtools_app/lib/src/debugger/debugger_controller.dart

```diff
@@ -38,7 +39,9 @@ class DebuggerController extends DisposableController
         .listen(switchToIsolate));
     autoDispose(_service.onDebugEvent.listen(_handleDebugEvent));
     autoDispose(_service.onIsolateEvent.listen(_handleIsolateEvent));
+    // TODO(kenz): do we want to listen with event history here?
     autoDispose(_service.onStdoutEvent.listen(_handleStdoutEvent));
+    // TODO(kenz): do we want to listen with event history here?
     autoDispose(_service.onStderrEvent.listen(_handleStderrEvent));
```

Comment:
@devoncarew @grouma - any input on whether we should listen to event history here? I would think listening with history would mean that the console would populate with stdout and stderr event that occurred before a user opens the DevTools debugger.

Reply:
From the web perspective we don't capture log messages until debugging is initiated so for consistency we wouldn't want event history. However I can see an argument to also display them if you have the capability.

Reply:
We should try to change that for the web. I'm pretty sure the chrome devtools protocol gives you an api to fetch all historic messages.  The old messages may only show up as plain text (helps them avoid memory leaks).